ddl/ddl_worker.go

```diff
@@ -101,6 +101,18 @@ func (d *ddl) getCheckOwnerTimeout(flag JobType) int64 {
 }
 
 func (d *ddl) checkOwner(t *meta.Meta, flag JobType) (*model.Owner, error) {
+	if NewOwnerChange {
```

Comment:
It‘s divided into two steps. 
The first step makes it correct. The second is remove this variable and update tests.